src/test/bin/quic_gtest.cpp

```diff
@@ -2375,12 +2381,14 @@ int main(int argc, char** argv) {
         } else if (strcmp("--duoNic", argv[i]) == 0) {
             UseDuoNic = true;
         } else if (strcmp("--useQTIP", argv[i]) == 0) {
-#if defined(QUIC_USE_RAW_DATAPATH) && defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
-            UseQTIP = true;
-#else
-            printf("QTIP is not supported in this build.\n");
-            return -1;
+            UseQTIP = CxPlatIsRawDatapath();
+#if !defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
```

Comment:
Maybe we expose a feature flag for QTIP support too.

Reply:
The flag is put in Datapath->Features, but at this point the flag (Datapath) is not available